frontend/src/metabase/visualizations/echarts/cartesian/model/series.ts

```diff
@@ -324,7 +324,7 @@ function shouldRenderCompact(
     );
   };
 
-  return getAvgLength(true) + 3 < getAvgLength(false);
+  return getAvgLength(true) + 2 < getAvgLength(false);
```

Comment:
Why do we want to change this threshold?

Reply:
I was playing around with it and accidentally committed it. I put in a commit to revert this change.